README.md

```diff
@@ -92,6 +92,14 @@ metadata:
   engine: true
 ```
 
+### RSpec Integration
+It's highly recommended that you add `--require stimpack/rspec` to your `.rspec` file to keep things simply.
```

Comment:
```suggestion
It's highly recommended that you add `--require stimpack/rspec` to your `.rspec` file to keep things simple.
```

Reply:
Also this is a big vague to me and something that might bother me a bit as a reader of this documentation. It seems like the "keep things simple" is more about allowing us to lean on the assumption that the user has done that to provide a simple internal implementation. If so, I don't care about that as a user of this and would just want to be told that I need to do this.

Reply:
Updated